src/main/java/controller/MainController.java

```diff
@@ -96,31 +96,39 @@ public static MainController getMainControllerInstance() {
 	}
 
 	/**
-	 * This method is used to initiate the button listener
+	 * Initialises the file upload frame and the button listeners
 	 */
 	public void init() {
 		PopupUploadFile uploadFile = new PopupUploadFile();
-		JButton import_button = uploadFile.getImportJButton();
-		initImportButtonAction(import_button, uploadFile);
+		JButton importButton = uploadFile.getImportJButton();
+		initImportButtonAction(importButton, uploadFile);
 	}
 
 	/**
-	 * This method is used to run the action of the Import Button.
+	 * Initialises the import button action listeners
```

Comment:
Shouldn't be "...action listener" instead of "...action listeners"?

Reply:
sure, I can change, I just thought action listeners sounds better to the ear

Reply:
```suggestion
	 * Initialises the import button action listener
```